src/core/thread/router_table.hpp

```diff
@@ -265,6 +265,22 @@ class RouterTable : public InstanceLocator
      */
     Router *GetRouter(const Mac::ExtAddress &aExtAddress);
 
+    /**
+     * This method returns if the router table contains a given `Neighbor` instance.
+     *
+     * @param[in]  aNeighbor  A reference to a `Neighbor`.
+     *
+     * @retval TRUE  if @p aNeighbor is a `Router` in the router table.
+     * @retval FALSE if @p aNeighbor is not a `Router` in the router table
+     *               (e.x. mParent, mParentCandidate, a `Child` of the child table).
```

Comment:
nit: `e.x.` -> `e.g.` or `i.e.`?
it seems `e.x.` only appear here or in `include/openthread/platform/otns.h`.

Reply:
fixed